flow-sample-intents/src/androidTest/java/flow/sample/basic/IntentsSampleTest.java

```diff
@@ -0,0 +1,42 @@
+package flow.sample.basic;
```

Comment:
@loganj None of the test classes or gradle files have copyright blocks. Is that intentional?